src/cli/commands/audit.js

```diff
@@ -238,7 +240,7 @@ export default class Audit {
 
   async _fetchAudit(auditTree: AuditTree): Object {
     let responseJson;
-    const registry = YARN_REGISTRY;
+    const registry = this.options.registry || YARN_REGISTRY;
```

Comment:
Even better, maybe add something like `|| String(this.config.getOption('registry')).replace(/\/+$/, '')` in the middle, so we also also consult the config file(s)? (_Credit to PR #6484 for that!_)